### PR TITLE
Fix BrowseSkill, integrated styled components, mobile view, Codacy Doc

### LINKED ISSUE #2219: CSS Bugs
**Actual Behaviour**
- There are a lot of CSS bugs in the app. Also, many elements are not responsive.
List of components - 

- [x] Auth related components
- [x] Settings page
- [ ] About pages
- [x] Details page
- [x] Skill home(BrowseSkill, etc)
- [ ] Skill creator and Edit pages
- [ ] Admin pages
- [x] Dashboard

Feel free to add more pages, if I missed.

**Expected Behaviour**
All these bugs should be fixed.

**Would you like to work on the issue?**
I, @fragm3 @harsh-jindal

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2219, #2223, #2247, #2195 

Changes: 
- [x] Update Codacy and Deploy to Docker on Readme
- [x] Fix Browseskill CSS bugs
- [x] CSS Cleanup
- [x] Integrated Styled components
- [x] Responsive website
- [x] Sideways scroll button not working correctly #2195
- [x] Remove left fab button if cards less than 5
- [x] Fix Category list in mobile view
- [x] Fix scroll button icon alignment

Demo Link: https://pr-2246-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot 2019-06-11 at 6 31 21 PM

Screenshot 2019-06-11 at 6 31 48 PM

Screenshot 2019-06-11 at 6 32 18 PM

Screenshot 2019-06-12 at 1 29 59 AM

Screenshot 2019-06-12 at 2 15 31 PM

Screenshot 2019-06-12 at 2 17 49 PM

### CODE REVIEW

File: src/components/cms/SkillCardGrid/SkillCardGrid.js [hunk 1]
```diff
@@ -2,17 +2,40 @@
 import React, { Component } from 'react';
 import Ratings from 'react-ratings-declarative';
 import { Link } from 'react-router-dom';
-import Card from '@material-ui/core/Card';
+// import Card from '@material-ui/core/Card';
```
Comment (harsh-jindal) [inline src/components/cms/SkillCardGrid/SkillCardGrid.js]: Remove this comment.

**Comment by harsh-jindal** [comment]
@fragm3 Fix this
Screenshot 2019-06-12 at 10 59 52 AM

**Comment by akshatnitd** [comment]
Also, the entire list doesn't gets scrolled.

**Comment by fragm3** [comment]
@harsh-jindal @akshatnitd fixed, please review. Added screenshot in PR description.

**Comment by fragm3** [comment]
@akshatnitd made the changes,

Screenshot 2019-06-13 at 1 48 56 AM
Screenshot 2019-06-13 at 1 49 08 AM

**Pull request merged**